src/main/scala/ScalaModulePlugin.scala

```diff
@@ -147,6 +147,17 @@ object ScalaModulePlugin extends AutoPlugin {
 
   lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = scalaModuleOsgiSettings
 
+  lazy val scalaModuleSettingsNative: Seq[Settings[_]] = Seq(
+    scalaVersion := "2.11.11",
+    skip in compile := System.getProperty("java.version").startsWith("1.6"),
+    test := {},
+    libraryDependencies := {
+      if (!scalaVersion.value.startsWith("2.11"))
+        libraryDependencies.value.filterNot(_.organization == "org.scala-native")
+      else libraryDependencies.value
+    }
```

Comment:
No, this is a workaround to the fact that ScalaNative plugins are going to be _in the classpath_ and SBT fails to resolve them when the Scala version is different than `2.11.x`. This is regardless of the fact that the previous settings and it's the only way I managed to make `scala-parser-combinators` cross-compile.

> the project should not even try to update/compile/test for such a Scala version

Of course!, I would love it if `sbt-crossproject` allowed to disable the _native build_, which is what this _hack_ packaged as `scalaModuleSettingsNative` tries to do so probably it's better to PR something less hacky in that other plugin. If you have any ideas I'd be happy to help.